auditbeat/module/file_integrity/action.go

```diff
@@ -47,6 +48,7 @@ var actionNames = map[Action]string{
 	Updated:            "updated",
 	Moved:              "moved",
 	ConfigChange:       "config_change",
+	Found:              "found",
```

Comment:
So this flag indicates that the event was generated due to the initial scan? If so how about we call it `initial_scan` rather than `found`?

Reply:
Makes sense - changed it.